### DISCUSSION TITLE
Any number on the outgoing route

### DISCUSSION THREAD

**Discussion by m4ak** [mikopbx/Core#587] [Q&A]
In the outgoing routing interface can be set such as X. T such a dialing scheme, MIKOPBX users can match this rule to send to the gateway.

I have now registered to the superior IPPBX device, the number on IPPBX is 3XXXXX, dial 0 to the carrier.

Want MIKOPBX users, dial 3XXXXX interworking, dial 0 dozen mobile phone numbers

**Comment by boffart** [comment]
see the table with examples
https://docs.mikopbx.com/mikopbx/v/english/manual/routing/outbound-routes#primery-shablonov-nomerov
